lib/rules/indent-function-args.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * @fileoverview Check for function args proper indentation.
+ * @author uberVU
+ * @copyright 2015 uberVU. All rights reserved.
+ * See LICENSE file in root directory for full license.
+ */
+'use strict';
+var getNodeIndent = require('../get-node-indent.js');
+var indentSize = 4;
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = function(context) {
+
+  var allignedWithParen = {
+        accepts: function(node, parent) {
+          /**
+           * Checks if argument is correctly alligned at parent parenthesis.
+           *
+           * @return {Boolean}
+           */
+          var argStartCol = node.loc.start.column,
+              functionEndCol = parent.callee.loc.end.column;
+
+          return argStartCol === functionEndCol + 1;
+        }
+
+      },
+      at4spacesFromFunctionCall = {
+        accepts: function(node, parent) {
+          /**
+           * Checks if argument is correctly alligned at indentSize from parent
+           * function call.
+           *
+           * @return {Boolean}
+           */
+          var argStartCol = node.loc.start.column,
+              functionStartCol = parent.loc.start.column;
+
+          return argStartCol === functionStartCol + indentSize;
+        }
+      },
+      at4spacesFromPrevLineIndent = {
+        accepts: function(node, parent) {
+          /**
+           * Checks if argument is correctly alligned at indentSize from prev
+           * line indent.
+           *
+           * @return {Boolean}
+           */
+          var argStartCol = node.loc.start.column,
+              functionIndent = getNodeIndent(parent, false, false, 'space',
+                  context);
+
+          return argStartCol === functionIndent + indentSize;
+        }
+      };
+
+  var indentModes = [
+    at4spacesFromFunctionCall,
+    at4spacesFromPrevLineIndent,
+    allignedWithParen
+  ];
+
+  function reportError(node, actualIndentation, requiredIndentation) {
```

Comment:
Noticed that it requires the `context` object, you can either pass it along or leave it nested here.